aiida_quantumespresso/calculations/pp.py

```diff
@@ -137,6 +139,9 @@ def prepare_for_submission(self, folder):  # pylint: disable=too-many-branches,t
         else:
             settings = {}
 
+        skip_plot = 'PLOT' not in parameters
+        copy_files = 'INPUTPP' not in parameters
```

Comment:
This is the criteria to decide what is happening: which namelist is missing from the parameters provided. I used these names since they are more related to the implications, but you can assume these logicals mean `step1_only` and `step2_only` respectively.